tests/images.bats

```diff
@@ -97,6 +97,19 @@ load helpers
   buildah rmi -a -f
 }
 
+@test "images json dup test" {
+  cid=$(buildah from scratch)
```

Comment:
(buildah from --signature-policy ${TESTSDIR}/policy.json

Reply:
For some reason it worked without locally, 39a7760 uses the policy.json

Reply:
Locally you have /etc/containers/policy.json installed, in travis the file does not exist.